src/PosteriorStats.jl

```diff
@@ -2,20 +2,31 @@ module PosteriorStats
 
 using Compat: @constprop
 using DataInterpolations: DataInterpolations
-using Distributions: Distributions
+using Distributions: cov, Distributions, invcov, mean
 using DocStringExtensions: FIELDS, FUNCTIONNAME, TYPEDEF, TYPEDFIELDS, SIGNATURES
 using FFTW: FFTW
-using IrrationalConstants: sqrthalfπ, sqrtπ, sqrt2
+using IrrationalConstants: sqrthalfπ, sqrtπ, sqrt2, log2π
 using IteratorInterfaceExtensions: IteratorInterfaceExtensions
 using KernelDensity: KernelDensity
-using LinearAlgebra: mul!, norm, normalize
+using LinearAlgebra:
+    cholesky,
+    Cholesky,
+    diag,
+    Diagonal,
+    LowerTriangular,
+    mul!,
+    norm,
+    normalize,
+    Symmetric,
+    UpperTriangular
```

Comment:
Only `diag` and `Diagonal` are used, and each only once, so maybe let's not import these either:
```suggestion
using LinearAlgebra: mul!, norm, normalize
```